svo (install the zip, not me).xml

```diff
@@ -54203,7 +54203,16 @@ end
 
 -- returns a list of affs that tree and other basic cures can get
 function svo.gettreeableaffs()
-  return table.n_intersection(svo.keystolist(svo.affl), svo.treecurables)
+  local a = table.n_intersection(svo.keystolist(svo.affl), svo.treecurables)
```

Comment:
Why not using the `svo.treecurableswithhypothermia` here instead of removing the others manually later?